<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>score_search</class>
 <widget class="QDialog" name="score_search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>成绩查询与统计</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="button_search">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#button_search{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#button_search:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#button_search:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}</string>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>1081</width>
     <height>591</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: qlineargradient(x1: 0, y1: 0, x2: 0.93, y2: 0.93,stop: 0 #1E90FF, stop: 1 white);</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked</set>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbb_findway">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;

    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}
QComboBox::down-arrow {
	image: url(:/image/image/icon/arrow_down.png);
}


QComboBox#cbb_findway{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,255);
}

QComboBox#cbb_findway:hover{
border-color: rgb(213, 198, 255);
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255,255);
}</string>
   </property>
   <item>
    <property name="text">
     <string>请选择查询方式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>所有成绩</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>姓名</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>学号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>班级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>科目</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="line_infor">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255, 255, 255,128);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入...</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>按分数段查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_lower">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255, 255, 255,128);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>最低...</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_upper">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255, 255, 255,128);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>最高...</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>~</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbb_subject">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;

    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}
QComboBox::down-arrow {
	image: url(:/image/image/icon/arrow_down.png);
}


QComboBox#cbb_subject{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,255);
}

QComboBox#cbb_subject:hover{
border-color: rgb(213, 198, 255);
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255,255);
}</string>
   </property>
   <item>
    <property name="text">
     <string>请选择查询科目</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="line_class">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255, 255, 255,128);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="placeholderText">
    <string>请输入查询班级...</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_average">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>670</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>null</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>670</y>
     <width>96</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>平均分：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>670</y>
     <width>96</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>及格率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_passrate">
   <property name="geometry">
    <rect>
     <x>527</x>
     <y>670</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>null</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>670</y>
     <width>96</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>标准差：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_standard_deviation">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>670</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>null</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
